theme/management/commands/migrate_published_resources.py

```diff
@@ -14,18 +40,19 @@ class Command(BaseCommand):
     help = "Migrate published resources to published user quota holder"
 
     def handle(self, *args, **options):
-        published_user = User.objects.get(username=settings.PUBLISHED_USER)
+        published_user = User.objects.get(username="published")
         resources = BaseResource.objects.filter(raccess__published=True).exclude(quota_holder=published_user)
         count = 0
         for res in resources:
             print('Fixing resource: {}'.format(res.short_id))
             try:
                 UserResourcePrivilege.share(user=published_user, resource=res,
                                             privilege=PrivilegeCodes.OWNER, grantor=res.quota_holder)
-                res.set_quota_holder(res.quota_holder, published_user)
+                set_quota_holder(res, published_user)
```

Comment:
gotta admit that I'm not seeing why you do this `set_quota_holder` instead of using `res.set_quota_holder(res.quota_holder, published_user)`

Reply:
I originally wrote it that way and I should have explained this. I spoke with Martin about the published resource migration and he suggested we copy the published resources to the new location without deleting them from the user buckets to give us some time to validate all the data was copied correctly before deletion (cleanup_migrated_published_resources) I had to copy the code and remove the deletion to accomplish this.

Reply:
GOT IT!